tests/TestCase.php

```diff
@@ -204,9 +204,9 @@ protected static function generateRandomString($length = 24)
     /**
      * Generate a semi-random email.
      */
-    protected static function generateRandomEmail($domain = 'bar.com')
+    protected static function generateRandomEmail()
     {
-        return self::generateRandomString().'@'.$domain;
+        return 'dev-platform-bots+php-'.self::generateRandomString(12).'@stripe.com';
```

Comment:
Arbitrarily chose a length of 12 because it's a lot of entropy without being ridiculously lengthy (default is 24).